biz.aQute.bndlib/src/aQute/bnd/signing/JartoolSigner.java

```diff
@@ -45,6 +45,12 @@
 		String sigFile() default "";
 
 		String digestalg() default "";
+
+		String tsaurl() default "";
```

Comment:
The other option names generally match the names of the corresponding jarsigner tool option. So this option should be `tsa`